k8s/bases/infrastructure/cluster-roles/tenant-edit.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tenant-edit
+aggregationRule:
+  clusterRoleSelectors:
+    - matchLabels:
+        devantler.tech/aggregate-to-tenant-edit: "true"
+# `rules` is intentionally OMITTED: the ClusterRole aggregation controller owns
+# and populates it from the labelled pieces. Declaring `rules: []` here would make
+# Flux's server-side apply fight the controller for that field on every reconcile.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)